dockers/docker-fpm-frr/frr/bgpd/templates/general/instance.conf.j2

```diff
@@ -15,13 +15,21 @@
 !
 {% if neighbor_addr | ipv4 %}
   address-family ipv4
+{% if bgp_session["asn"] == bgp_asn %}
+    neighbor {{ neighbor_addr }} peer-group PEER_V4_INT
+{% else %}
     neighbor {{ neighbor_addr }} peer-group PEER_V4
+{%   endif %}
 {%   if CONFIG_DB__DEVICE_METADATA['localhost']['sub_role'] == 'BackEnd' %}
     neighbor {{ neighbor_addr }} route-map FROM_BGP_PEER_V4_INT in
 {%   endif %}
 {% elif neighbor_addr | ipv6 %}
   address-family ipv6
+{% if bgp_session["asn"] == bgp_asn %}
```

Comment:
why is this not consistent with line 45?

Reply:
Sure, I have updated here too to check "ASIC" keyword to identify internal sessions. thanks.